libraries/core/src/test/java/com/paritytrading/philadelphia/FIXInitiatorTestWithSocketChannel.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 Philadelphia authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.paritytrading.philadelphia;
+
+import java.io.IOException;
+import java.nio.channels.ServerSocketChannel;
+import java.nio.channels.SocketChannel;
+
+class FIXInitiatorTestWithSocketChannel extends FIXInitiatorTest<SocketChannel> {
```

Comment:
You were absolutely right about the naming conventions; JUnit 5 does not actually pick these tests up anymore on `mvn test`. 🤦🏼  Would you mind renaming these to `FIXInitiatorPipeChannelTest` and `FIXInitiatorTestSocketChannelTest` after all? Sorry! 😅

Reply:
Thanks, done.

For the record, this is a Maven and not a JUnit issue (IntelliJ and Gradle can pick these just fine). We can override the pattern of the SureFire plugin, but it is generally a bad idea with Maven.